app/scripts/lib/router.js

```diff
@@ -129,6 +129,7 @@ function (
       'confirm_reset_password(/)': showView(ConfirmResetPasswordView),
       'cookies_disabled(/)': showView(CookiesDisabledView),
       'force_auth(/)': showView(ForceAuthView),
+      'force_auth_complete(/)': showView(ReadyView, { type: 'force_auth' }),
```

Comment:
this view shows "Your account is ready!" no matter what (with or without existing sessions)

Do we need do any checking / error reporting or is that by design?